packages/esm-ohri-pmtct-app/src/views/child-health/tabs/infant-postnatal-care.component.tsx

```diff
@@ -25,7 +25,7 @@ interface InfantPostnatalListProps {
 
 const InfantPostnatalList: React.FC<InfantPostnatalListProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
-  const headerTitle = t('infantPostnatalCare', 'Infant Postnatal Care');
+  const headerTitle = t('HIVExposedInfant', 'HIV Exposed Infant');
```

Comment:
Change the translation key to `hivExposedInfant` to match with what is in the translations

Reply:
I have made new commit.